tmdata/genesis.json

```diff
@@ -0,0 +1 @@
+{"genesis_time":"0001-01-01T00:00:00Z","chain_id":"test-chain-EhS6zg","validators":[{"pub_key":{"type":"ed25519","data":"C3E96823EB67401C5B794F4100CEA04B745C29A0707979485EAE4F3C1A7D8583"},"power":10,"name":""}],"app_hash":""}
```

Comment:
what is this file for in addition to genesis files above?

Reply:
That is for the standalone setup.